view-and-share/tutorial.md

```diff
@@ -0,0 +1,235 @@
+
+# Pryv View and share data tutorial
+
+All you need to run this app is to download [index.html](index.html) and [script.js](script.js) files and open **index.html** with your browser.
+
+This is a visualization and sharing web app that first displays a welcome message and a button to initiate the authentication process.
+<p align="center">
+<img src="images/login.png" alt="login" />
+</p>
+
+
+With click on the login button, a popup opens in your browser where you can either authenticate or create a new account. 
+
+When signed in, you can consent to give the app access to the streams "Body" and "Baby" where the data from the form (see [tutorial "Collect survey data"](collect-survey-data/tutorial.md)) is stored.
+<p align="center">
+<img src="images/request-permissions.png" alt="request-permissions" width=250 />
+</p>
+Once you have accepted, it displays the saved data of your baby's weight and your own blood pressure.
```

Comment:
maybe add a small view with displayed data, from an account that has 1-2 entries of each so it fits the guide conveniently?